internal/pkg/service/buffer/file/file_create.go

```diff
@@ -0,0 +1,72 @@
+package file
+
+import (
+	"context"
+	"fmt"
+	"sync"
+
+	"github.com/keboola/go-client/pkg/keboola"
+
+	"github.com/keboola/keboola-as-code/internal/pkg/service/buffer/store/model"
+	"github.com/keboola/keboola-as-code/internal/pkg/service/common/rollback"
+	"github.com/keboola/keboola-as-code/internal/pkg/utils/errors"
+)
+
+func (m *Manager) CreateFilesForReceiver(ctx context.Context, rb rollback.Builder, receiver *model.Receiver) error {
+	rb = rb.AddParallel()
```

Comment:
V tomto commite som popresuval kod, aby boli operacie nad suborov, vo `buffer/file` pkg.

Uz sme tam mali predtym `buffer/storage/file`, ale nebolo tam vsetko.

Teraz som sem doplnil operacie `swapSlice` a `swapFile`.

V buducnosti chcem mat vsetky operacie nad subormi tu, toto je prvy krok.

Reply:
Ono dává smysl pak ty pkg dělit interně trochu podle DDD, v connection používáme přibližně
```
/module
-> /pkg
  -> /action
    -> bussiness-logic
    -> input-objects
    -> output-objects
    -> security
  -> /model
  -> /security
  -> /events
  -> common
```
Náš problém je teda, že celkově doména stojí dost za hovno, takže model spíš neexistuje a násobí sa to x200 endpointů x15let x50cli commandů :) 

V go to možná není potřeba zas tak detailně dělit, protože modelové struktury sa často cpů různě do akcí, není to tak striktní jak v PHP. Ale uvidíš kam to naroste :)

Reply:
Co mě přijde ještě hodně divné, že máš Manager struct a metody definuješ mimo ten manager ve vlastním souboru je to jako bych si v PHP udělal class a kolem navalil traity abych logicky oddělil funkce.
Jakože nic proti, ale přišlo by mě "kognitivně" přívětivější kdyby to byl `manager_file_create.go` jakože rozšiřuješ funkce té základní struct.